Add REVIEWING.md (#15)
security-summer-school/template-public: REVIEWING.md

> # Reviewing contributed session

Reviewer: ```suggestion
# Reviewing a Modified / Newly Added Session
```

## Activities / Challenges

> You will make sure that activities / challenges are of good quality:

Reviewer: ```suggestion
Make sure that activities / challenges are of good quality:
```
Let's keep sentencens imperative as opposed to future. They are more compelling like this.

- Difficulty is properly set.
- The topic of the activity / challenge is appropriate.

> - The solution provided is working.

Reviewer: ```suggestion
- The provided solution is working.
```

- Vulnerability and explanations are easy to understand.

## Assets

> For assets, you need to check if all of them are used and mentioned in the content and make sure that the name is appropriate.

Reviewer: ```suggestion
Check if all of them are used and mentioned in the content.
If the assets are not original, make sure their license is compatible to [ours](./COPYING.md) and that their source is properly cited.
Verify that their names are appropriate.
```

Let's settle on a naming convention. What about `image-name.png`, i.e. using a dash between words, as opposed to an underscore, for example.

Author: I didn't commit this suggestion since I added also some rules.
Can you please check them and mark this as solved if you think so?

## Content

> You will make sure that the content is of good quality:

Reviewer: ```suggestion
Make sure that the content is of good quality:
```

- Follow the general structure of a [session](session-name-1/README.md).
- Assets are displayed well.
- The content should have at most one sentence per line.
- The content is easy to understand.

## Working on Pull Requests

You will review contributions as pull requests.

> Contributions will need to follow the [contributing guidelines](CONTRIBUTING.md).

Reviewer: ```suggestion
Contributions must follow the [contributing guidelines](CONTRIBUTING.md).
```

> Add your review as comments and request for changes to the PR until everything is ready to be approved.

Reviewer: ```suggestion
Add your review as comments and request for changes to the PR until everything is ready to be approved.
Add any comments regarding one or more explicit lines as review comments and only publish them together when submitting your review.
Do not add each individual suggestion or comment as standalone.
```

> # Reviewing a Modified / Newly Added Session

Reviewer 2: ```suggestion
# SSS Reviewer Guidelines

These are recommendations for reviewing contributions to SSS repositories.
Reviewing is important to ensure high quality and consistency of content.
As a reviewer, we trust you will vet contributions and provide constructive suggestions to update them to fit the SSS guidelines.
```

## Activities / Challenges

Make sure that activities / challenges are of good quality:

- Difficulty is properly set.
- The topic of the activity / challenge is appropriate.
- The provided solution is working.
- Vulnerability and explanations are easy to understand.

## Assets

Check if all of them are used and mentioned in the content.
If the assets are not original, make sure their license is compatible to [ours](./COPYING.md) and that their source is properly cited.
Verify that their names are appropriate and respect the following rules:

> - Use dash between words(e.g. `image-name.png`).

Reviewer 2: ```suggestion
- Use dash between words (e.g. `image-name.png`).
```

> - Use only word characters.

Reviewer: ```suggestion
- Use only lowercase ASCII characters.
```

## Content

Make sure that the content is of good quality:

> - Follow the general structure of a [session](session-name-1/README.md).

Reviewer: ```suggestion
- It follows the general structure of a [session](session-name-1/README.md).
```

- Assets are displayed well.

> - The content should have at most one sentence per line.

Reviewer: ```suggestion
- The content has at most one sentence per line.
```

> - The content is easy to understand.

Reviewer 2: ```suggestion
- The content is easy to understand.
- Wording and general spelling are correct.
- Proper formatting is used (such as bold, italic, typewritter formats).
```

## Working on Pull Requests

> You will review contributions as pull requests.

Reviewer 2: ```suggestion
You will review contributions as pull requests, following the [PR review guidelines from GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews).
```
